README.md

```diff
@@ -532,19 +527,19 @@ through `protoc` using one of 2 patterns:
 - as part of the `--<tool_suffix>_out` `protoc` parameter: `--<tool_suffix>_out=<flags>:<path>`
 
 ```sh
---grpc-gateway_out=logtostderr=true,repeated_path_param_separator=ssv:.
---openapiv2_out=logtostderr=true,repeated_path_param_separator=ssv:.
+--grpc-gateway_out=repeated_path_param_separator=ssv:.
+--openapiv2_out=repeated_path_param_separator=ssv:.
 ```
 
 - using additional `--<tool_suffix>_opt` parameters: `--<tool_suffix>_opt=<flag>[,<flag>]*`
 
 ```sh
---grpc-gateway_opt logtostderr=true,repeated_path_param_separator=ssv
+--grpc-gateway_opt param1=value1,repeated_path_param_separator=ssv
 # or separately
---grpc-gateway_opt logtostderr=true --grpc-gateway_opt repeated_path_param_separator=ssv
---openapiv2_opt logtostderr=true,repeated_path_param_separator=ssv
+--grpc-gateway_opt param1=value1 --grpc-gateway_opt repeated_path_param_separator=ssv
+--openapiv2_opt param1=value1,repeated_path_param_separator=ssv
 # or separately
---openapiv2_opt logtostderr=true --openapiv2_opt repeated_path_param_separator=ssv
+--openapiv2_opt param1=value1 --openapiv2_opt repeated_path_param_separator=ssv
```

Comment:
Thank you for removing the old invocations, but I think we can just remove these example invocations too (the `param1=value1` stuff).

Reply:
Done